libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -511,11 +513,11 @@ def update_sol_yaml(self, worker_obj: list, filepath: str, cortx_image: str,
             # The condition to validate the config.
             if not skip_disk_count_check and valid_disk_count > \
                     (cvg_count * node_list):
-                return False, "The sum of data disks per cvg " \
+                assert False, "The sum of data disks per cvg " \
```

Comment:
Do we want to assert in lib ?

Reply:
as the error message was very generic and we are not explicitly using this method in our test file so using assert to capture correct error message.